File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UnitsOfMeasure.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class UnitsOfMeasure {
    @JsonProperty("Abbreviation")
    public String abbreviation;
    @JsonProperty("Name")
    public String name;
    @JsonProperty("QuantityType")
    public String quantityType;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/UnitsOfMeasure.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class UnitsOfMeasure(
    @JsonProperty("Abbreviation")
    val abbreviation: String? = null,
    @JsonProperty("Name")
    val name: String? = null,
    @JsonProperty("QuantityType")
    val quantityType: String? = null
)
